<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWidget</class>
 <widget class="QWidget" name="QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Do you want to make single Bid or multiple bid?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="YesButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NoButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome User</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>691</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLineEdit" name="Price1">
      <property name="placeholderText">
       <string>Price1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Quantity1">
      <property name="placeholderText">
       <string>Quantity1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="From1">
      <property name="placeholderText">
       <string>From Hour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="To1">
      <property name="placeholderText">
       <string>To Hour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="Date1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>691</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLineEdit" name="Price2">
      <property name="placeholderText">
       <string>Price2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Quantity2">
      <property name="placeholderText">
       <string>Quantity2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="From2">
      <property name="placeholderText">
       <string>From Hour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="To2">
      <property name="placeholderText">
       <string>To Hour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="Date2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>691</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLineEdit" name="Price3">
      <property name="placeholderText">
       <string>Price3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Quantity3">
      <property name="placeholderText">
       <string>Quantity3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="From3">
      <property name="placeholderText">
       <string>From Hour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="To3">
      <property name="placeholderText">
       <string>To Hour</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="Date3"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="FinalButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalize the Bid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>* Once changed you cannot edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SeeResultButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>410</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>See the Result</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
